Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM python:3.10
+FROM python:3.12-slim-bookworm
+# RUN apt update && apt install make
+RUN apt update && apt upgrade && apt install make -y
 COPY . /app
 WORKDIR /app
 RUN make clean install
+ENTRYPOINT []
```

Comment:
why an empty entrypoint?

Reply:
Actually just for consistency - `.wolfi` file has empty entry point, so I just added it here. In practice it does not change anything